### PR TITLE
Add units of measure and deactivate taxes

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

File: configuration/initializer_config.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "models" : [
+     
+        {
+            "model_name": "syncable.units.mapping",
+            "folder": "syncable_units_mapping",
+            "field_rules": {}
+        },
+        {
+            "model_name": "product.template",
```
Comment (rbuisson) [inline configuration/initializer_config.json:10]: Is this to load the drugs?
If so, the default Drug Loader now supports loading costs and uoms

File: configuration/units_of_measure/product.uom.csv [hunk 2]
```diff
@@ -0,0 +1,20 @@
+category_id/id,factor,rounding,uom_type,id,name
+product.product_uom_categ_vol,1,0.01,Reference Unit of Measure for this category,f303ec09-ab78-43d9-abf2-06839f4eb491,Ampule(s)
```
Comment (rbuisson) [inline configuration/units_of_measure/product.uom.csv:2]: This should probably be set in 'fr'

File: configuration/syncable_units_mapping/syncable.units.mapping.csv [hunk 3]
```diff
@@ -0,0 +1,20 @@
+id,name,unit_of_measure/id
+syncable_units_Ampule(s),Ampule(s),f303ec09-ab78-43d9-abf2-06839f4eb491
```
Comment (rbuisson) [inline configuration/syncable_units_mapping/syncable.units.mapping.csv:2]: This should probably be set in 'fr'

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: configuration/initializer_config.json (hunk 1 above)
Comment (reagan-meant) [reply configuration/initializer_config.json:10]: Removed

**Review by rbuisson — APPROVED** [review]

**Pull request merged**